README.md

```diff
@@ -26,10 +26,10 @@ This package provides a KeyShot plugin script that creates jobs for AWS Deadline
 ### Installing and using the KeyShot Submitter
 
 1. Run `pip install deadline[gui]`
-2. Copy the file `deadline-cloud-for-keyshot/keyshot_submitter/Submit to AWS Deadline Cloud.py` to the KeyShot scripts folder for your OS:
+2. Copy the file `deadline-cloud-for-keyshot/src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py` to the KeyShot scripts folder for your OS:
```

Comment:
Are we sure that the instructions are actually wrong? 

This is now assuming you're git cloning, but the instructions right above say `pip install deadline[gui]`. So I'm thinking that there's two potential workflows getting criss-crossed. There's a developer workflow with a local checkout, and then there's a user flow that installing releases.

My suggestion is that we make those distinct. README.md should probably have user instructions, DEVELOPER.md should probably have developer instructions. Thoughts?

Reply:
Because `deadline-cloud-for-keyshot` has not been released since June 19th, `pip install deadline-cloud-for-keyshot "deadline[gui]"` does not yet contain the latest version of the submitter and the instructions would be incorrect. That's why we still have these additional instructions for both users and developers. When the next release of `deadline-cloud-for-keyshot` occurs, we can update these instructions.

Reply:
right, sounds like we need to release soon ;)